libraries/AP_Param/AP_Param.h

```diff
@@ -281,13 +286,24 @@ class AP_Param
     /// @param  value           The new value
     /// @return                 true if the variable is found
     static bool set_by_name(const char *name, float value);
+    // name helper for scripting
+    static bool set(const char *name, float value) { return set_by_name(name, value); };
```

Comment:
These helpers actually shrink the build in my test. The compiler is smart enough to just point to the same implementation, and it saves us the string space in the script code. We could just rename `set_by_name` in the whole code base, which I'd be fine with, but it would have been a lot of churn on this commit, and made this drag on more, so I advised this approach.